apps/pxweb2/src/app/components/ContentTop/ContentTop.tsx

```diff
@@ -0,0 +1,52 @@
+import cl from 'clsx';
+
+import { useTranslation } from 'react-i18next';
```

Comment:
The blank line here seems to be on the wrong line. Maybe move the import from 'react-i18next' to line 2?